src/templates/fr/2019/methodology.html

```diff
@@ -0,0 +1,512 @@
+            <td>Moto G4 Émulé</td>
+          </tr>
+
+          <tr>
+            <td>User Agent</td>
+            <td>
+              Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.100 Safari/537.36 PTST/190704.170731
+            </td>
+            <td>
+              Mozilla/5.0 (Linux; Android 6.0.1; Moto G (4) Build/MPJ24.139-64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.146 Mobile Safari/537.36 PTST/190628.140653
+            </td>
+          </tr>
+
+          <tr>
+            <td>Emplacement</td>
+            <td>
+              Redwood City, Californie, États-Unis<br />
+              The Dalles, Oregon, États-Unis
+            </td>
+            <td>
+              Redwood City, Californie, États-Unis<br />
+              The Dalles, Oregon, États-Unis
+            </td>
+          </tr>
+
+          <tr>
+            <td>Connexion</td>
+            <td>Cable (5/1&nbsp;Mbps 28&nbsp;ms de latence)</td>
+            <td>3G (1.600/0.768&nbsp;Mbps 300&nbsp;ms de latence)</td>
+          </tr>
+
+          <tr>
+            <td>Viewport</td>
+            <td>1200&nbsp;×&nbsp;1920&nbsp;px</td>
+            <td>512&nbsp;×&nbsp;360&nbsp;px</td>
+          </tr>
+        </tbody>
+      </table>
+      </div></div>
+
+      <p>
+        Les sites Web pour ordinateurs de bureau sont exécutés à partir d’un environnement de bureau Chrome sur une VM Linux. La vitesse du réseau est équivalente à celle d’une connexion par câble.
+      </p>
+
+      <p>
+        Les sites Web mobiles sont visités à partir d’un environnement Chrome mobile sur un appareil Moto G4 émulé avec une vitesse réseau  équivalente à une connexion 3G. Notez que le User Agent mobile émulé s’identifie comme étant Chrome 65 mais est en réalité, Chrome 75.
+      </p>
+
+      <p>
+        Les tests sont effectués depuis deux localisations américaines&nbsp;:  en Californie et en Oregon. HTTP Archive maintient l’ensemble des matériels servant aux tests au sein du datacenter  <a href="https://archive.org">Internet Archive</a> situé en Californie.  Si nécessaire des périphériques de test supplémentaires sont ajoutés depuis  <a href="https://cloud.google.com/compute/docs/regions-zones/#locations" >Google Cloud Platform</a >, emplacement us-west-1, en Oregon.
+      </p>
+
+      <p>
+        L’instance privée de WebPageTest de HTTP Archive est synchronisée avec la dernière version publique et complétée par des <a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics" >indicateurs personnalisés</a >. Ils sont collectés par des portions de JavaScript qui sont évaluées et exécutées sur chaque site web à la fin du test. Le script sur-mesure  <a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js" >almanac.js</a > collecte plusieurs métriques qui sont autrement impossibles à calculer comme, par exemple,  celles qui dépendent de l'état DOM.
```

Comment:
> de l'état DOM

De l'état du DOM

Reply:
FYI during a review you can "suggest" changes, and the author can accept your suggestions directly in GitHub. Not that I needed that, but I share the info in case you missed it.

Reply:
Yes, I was hesitating to use this feature ^^

Reply:
I’m a big fan of this feature so, as far as I’m concerned, use away.

Saying that if you have to use it loads for lots of the same edits then better to comment and get the PR author to submit one more commit with all of them in one go.